trento/migration/container-install.md

```diff
@@ -63,13 +63,23 @@ Follow the steps in [4.2 systemd deployment](https://documentation.suse.com/sles
    REFRESH_TOKEN_ENC_SECRET=$(openssl rand -out /dev/stdout 48 | base64)
    ```
 
+1. Install the checks on the system in a shared volume
+
+   ```bash
+   docker volume create trento-checks \
+     && docker run \
+     -v trento-checks:/usr/share/trento/checks \
+     registry.suse.com/trento/trento-checks:1.0.0
+   ```
+
 1. Install trento-wanda on Docker:
```

Comment:
The sentence sounds weird: you can't install something **on** Docker. Do you mean something like, _Run trento-wanda in a Docker container_?

Reply:
@dmpop I am not familiar enough with Docker to argue otherwise. If you think your suggestion is more accurate, then go for it.

Reply:
"Install trento-wanda on Docker" changed to "Deploy trento-wanda" in both MD and XML files.
"Install trento-web on Docker" changed to "Deploy trento-web" in both MD and XML files.